### DISCUSSION TITLE
is there anyway to capture a scrolling screenshot

### DISCUSSION THREAD

**Discussion by lucy7099** [percy/cli#546] [Support] [answered]
is there anyway to capture a scrolling screenshot using the Percy CLI snapshot command? I have a snapshot.yml file but not sure if its possible. 

at the moment it just takes a screenshot of the page but not a scrolling one and cuts off everything else at the bottom. 

thanks

**Comment by Robdel12** [comment, marked as answer]
Hey @lucy7099! Percy captures full page screenshots by default. If the screenshot only contains the viewport, it's likely you have CSS on your page that clips the overflow to the content container. All browsers will respect this CSS when generating a screenshot and only capture the visible portion of the viewport. You can pass Percy CSS to unset the overflow on the containing div and it should result in a full page screenshot.
